packages/office-ui-fabric-react/src/components/Image/Image.tsx

```diff
@@ -91,7 +92,7 @@ export class Image extends React.Component<IImageProps, IImageState> {
 
     // If image dimensions aren't specified, the natural size of the image is used.
     return (
-      <div className={ css('ms-Image', className) } style={ { width: width, height: height } }>
+      <div className={ css('ms-Image', className) } style={ { width: width, height: height } } ref='frame'>
```

Comment:
You might want to have Image extend BaseComponent and use this._resolveRef('frame') for more safety. Probably not necessary right now though.